### ISSUE TITLE
Unable to register an account

### ISSUE THREAD

**Issue by loycoder** [steel-dev/steel-browser#20]
Through https://app.steel.dev/sign-up, enter, registered accounts, eventually prompt:
Sign up unsuccessful due to failed security validations. Please refresh the page to try again or reach out to support for more assistance.
<img width="526" alt="image" src="https://github.com/user-attachments/assets/6166c560-79fe-42ad-98cc-de41e63fae4d">

**Comment by fukouda** [comment]
Thats weird-- gonna look into this.

In the meantine, you should join the discord as we'll mostly be dealing with issues for the open source repo here

**Comment by hussufo** [comment]
hey @loycoder - sorry this is happening to you! Our auth is managed by clerk so it's pretty tricky to debug. Would you be able to try to sign in using a different method or, if you ping us in discord, we can get you set-up with an account manually.

**Comment by Mike7-777** [comment]
Yes, I'm getting this too. It must deter a lot of people. I am trying to sign in through Google but I have tried it via just email too. No luck!! Very frustrating :(

**Comment by hussufo** [comment]
Hey @Mike7-777  could you share your email or add me on discord @hussufo if you'd like to stay private? This should've been resolved but we can get your account sorted out.
